layouts/shortcodes/logos.html

```diff
@@ -2,6 +2,9 @@
   <div class="mt-4 col col-md-4 col-6">
     <a href="https://akamai.com" target="_blank" rel="noopener"><img src="/img/logos/akamai.png" title="Akamai" /></a>
   </div>
+  <div class="mt-4 col col-md-4 col-6">
+    <a href="https://www.bmc.com/" target="_blank" rel="noopener"><img src="/img/logos/static/BMC.png" title="BMC" /></a>
```

Comment:
The link here is incorrect it should be 

```suggestion
<a href="https://www.bmc.com/" target="_blank" rel="noopener"><img src="/img/logos/BMC.png" title="BMC" /></a>
```